hbase-shaded/hbase-shaded-testing-util/pom.xml

```diff
@@ -175,4 +151,81 @@
       </plugin>
     </plugins>
   </build>
+  <profiles>
+    <profile>
+      <id>hadoop-2.0</id>
+      <activation>
+        <property>
+          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
```

Comment:
Heya @saintstack -- this comment is about some support scripts, but I don't see them in the commit. Maybe something you intended to commit with this patch but missed the `git add` ? Thanks boss.

Reply:
Thanks for taking a looksee @ndimiduk 

It looks like this is a blind copy/paste of hadoop-2.0 profile activation snippet. I see we are repeating the comment in lots of places (See below). The comment is way stale referring to a script that was removed in 2014.

```
commit 37654f77a60ad0c13a84d6d6b39657789726df09
Author: Michael Stack <stack@apache.org>
Date:   Wed Mar 19 23:08:55 2014 +0000

    HBASE-10781 Remove hadoop-one-compat module and all references to hadoop1

    git-svn-id: https://svn.apache.org/repos/asf/hbase/trunk@1579449 13f79535-47bb-0310-9956-ffa450edef68

```

Here is grep for comment over code base:

```
$ grep -r 'Below formatt' .
./hbase-client/pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
./hbase-asyncfs/pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
./hbase-replication/pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
./hbase-hadoop2-compat/pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
./pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
./hbase-zookeeper/pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
./hbase-thrift/pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
./hbase-common/pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
./hbase-examples/pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
./hbase-procedure/pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
./hbase-endpoint/pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
./hbase-testing-util/pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
./hbase-server/pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
./hbase-rest/pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
./hbase-shaded/hbase-shaded-client-byo-hadoop/pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
./hbase-shaded/hbase-shaded-testing-util/pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
./hbase-shaded/hbase-shaded-mapreduce/pom.xml:          <!--Below formatting for dev-support/generate-hadoopX-poms.sh-->
....

```

Let me file an issue to clean out the comment... HBASE-27396

Reply:
Oh okay. Here I was hoping that there was some additional secret sauce I was missing.